make/buf/all.mk

```diff
@@ -144,6 +144,9 @@ bufgeneratebuflinttestdata:
 	$(BUF_BIN) export \
 		buf.build/bufbuild/protovalidate:$(PROTOVALIDATE_VERSION) \
 		--output private/bufpkg/bufcheck/testdata/lint/protovalidate/vendor/protovalidate
+	$(BUF_BIN) export \
```

Comment:
Why are we doing this? This appears to be exactly the same thing as in the above directory.